jdaviz/core/template_mixin.py

```diff
@@ -1517,12 +1523,24 @@ def selected_obj(self):
 
     @property
     def selected_subset_state(self):
+        if self.is_multiselect:
+            subset_states = {}
```

Comment:
I don't think other instances of multiselect return dictionaries... but since this is not terribly public-facing, this is probably ok for now and we can consider later whether to make everything dictionaries or lists.